src/components/oba/StopPane.svelte

```diff
@@ -91,27 +92,7 @@
 
 <div>
 	{#if loading}
-		<div
-			class="absolute inset-0 z-10 flex items-center justify-center rounded-lg bg-[#1C1C1E] bg-opacity-80"
-		>
-			<div class="flex items-center text-white">
-				<svg
-					class="-ml-1 mr-3 h-5 w-5 animate-spin text-white"
-					xmlns="http://www.w3.org/2000/svg"
-					fill="none"
-					viewBox="0 0 24 24"
-				>
-					<circle class="opacity-25" cx="12" cy="12" r="10" stroke="currentColor" stroke-width="4"
-					></circle>
-					<path
-						class="opacity-75"
-						fill="currentColor"
-						d="M4 12a8 8 0 018-8V0C5.373 0 0 5.373 0 12h4zm2 5.291A7.962 7.962 0 014 12H0c0 3.042 1.135 5.824 3 7.938l3-2.647z"
-					></path>
-				</svg>
-				{$t('loading')}...
-			</div>
-		</div>
+		<LoadingSpinner />
```

Comment:
👍

Reply:
Hi @aaronbrethorst 

The search bar is working but before I PR, I also added search by rotueId, I think I will remove it from the search filter so we can search by short and long name and description only